### DISCUSSION TITLE
Feature multiplication

### DISCUSSION THREAD

**Discussion by Samaeldll** [azerothcore/azerothcore-wotlk#11872] [General]
Welcome, please tell me how to fix it, or maybe the direction in which you need to dig.
there are spells that increase performance in% ratio, or damage by 10-20-30. %.
the problem is that recently these percentages have been multiplying, that is, 5% + 5% = 25%.
where you can correct this formula, or, for example, add a coefficient to it. I can’t find the location of this formula in the kernel, can someone tell me where to find it, or is it possible to solve this problem?
------------------------------------------------------------------------------------------------------------------------------------------
ru: Приветствую, подскажите пожалуйста как исправить, а может направление в каком нужно копать.
есть заклинания, повышающие характеристики в % соотношении, или урон на 10-20-30. %.
проблема в том, что в последнее время эти проценты умножаются, то есть 5%+5%=25%.
где можно исправить эту формулу, или, например, добавить к ней коэффициент. не могу найти расположение этой формулы в ядре, может кто подскажет где найти, или возможно ли решить эту проблему?

**Comment by Winfidonarleyan** [comment]
В этом нет ошибки, так и должно быть
